framework/source/class/qx/ui/core/LayoutItem.js

```diff
@@ -516,6 +496,29 @@ qx.Class.define("qx.ui.core.LayoutItem",
         delete this.__updateMargin;
       }
 
+      /*
+       * Height for width support
+       * 
+       * Results into a re-layout which means that width/height is applied in the next iteration.
+       * 
+       * Note that it is important that this happens after the above first pass at calculating a 
+       * computed size because otherwise getBounds() will return null, and this will cause an
+       * issue where the existing size is expected to have already been applied by the layout.
+       * See https://github.com/qooxdoo/qooxdoo/issues/9553  
+       */
+      if (this.getHeight() == null && this._hasHeightForWidth()) {
+        var flowHeight = this._getHeightForWidth(width);
+        
+        if (flowHeight != null && flowHeight !== this.__computedHeightForWidth)
```

Comment:
`!=` with null... Are you checking for falsy, or do you mean to check explicitly for null?

Reply:
That code is just moved down - I agree that the comparisons are suspect, but it dates back to before 2011, so I've just kept it as it is